CONTRIBUTING.md

```diff
@@ -64,12 +64,16 @@ Our commit style is defined in a __[commit template]__. Use it as a reference or
 ### Rules and Recommendations
 
 - *Use your __real name__ and __real email__.* We do not accept anonymous contributions!
+  - Every commit should have author's full legal name (in latin letters, diacritics allowed) with real e-mail address.
```

Comment:
As it currently stands, this rule regarding the exposure of your real name is optional for people such as artists and translators whereas your wording of this PR makes the said rule compulsory for everyone. That's too rigorous and I'm not so keen about it (plus we had someone who ranted about it on Twitter sometime ago but I would rather not want go into specifics).

I would suggest that you at least make such rule mandatory only for people who contribute with code to the project and if someone is too paranoid to use their legal name for contribution due to understandable reasons such as privacy, we should adopt the approach of accepting user aliases where a contributor signs a legal document (which should include their real name for legal reasons obviously) so they can use an alias that is easily recognizable so no stupid crap nicknames like `trololololo` or `Xxx CykaBloat xxX`. The ReactOS project is fully responsible to not disclose such document at this point.

Reply:
Such thing as "optional for people such as artists and translators" is actually documented nowhere. I believe it was in an open discussion without the final decision, and that is why current contributing rules do not include it and just require the real name.

> plus we had someone who ranted about it on Twitter sometime ago

If I remember correctly, that was because PR was merged, but we required the full name from a person only after that. Good reason to make a final decision on that, to avoid these rants in future.

Reply:
I have added an exception point for the artists (changes related to wallpapers, themes, icons, sounds), please review. 🙂